### ISSUE TITLE
2025-05-12, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#167]
# Today's Arxiv Papers

(1) [From Millions of Tweets to Actionable Insights: Leveraging LLMs for User Profiling](http://arxiv.org/abs/2505.06184v1), 2025-05-09

(2) [A Large Language Model-Enhanced Q-learning for Capacitated Vehicle Routing Problem with Time Windows](http://arxiv.org/abs/2505.06178v1), 2025-05-09

(3) [MonetGPT: Solving Puzzles Enhances MLLMs' Image Retouching Skills](http://dx.doi.org/10.1145/3730926), 2025-05-09

(4) [Turbo-ICL: In-Context Learning-Based Turbo Equalization](http://arxiv.org/abs/2505.06175v1), 2025-05-09

(5) [A Scaling Law for Token Efficiency in LLM Fine-Tuning Under Fixed Compute Budgets](http://arxiv.org/abs/2505.06150v1), 2025-05-09

(6) [Can Prompting LLMs Unlock Hate Speech Detection across Languages? A Zero-shot and Few-shot Study](http://arxiv.org/abs/2505.06149v1), 2025-05-09

(7) [ELA-ZSON: Efficient Layout-Aware Zero-Shot Object Navigation Agent with Hierarchical Planning](http://arxiv.org/abs/2505.06131v1), 2025-05-09

(8) [LLMs Get Lost In Multi-Turn Conversation](http://arxiv.org/abs/2505.06120v1), 2025-05-09

(9) [LLMs Outperform Experts on Challenging Biology Benchmarks](http://arxiv.org/abs/2505.06108v1), 2025-05-09

(10) [Free and Fair Hardware: A Pathway to Copyright Infringement-Free Verilog Generation using LLMs](http://arxiv.org/abs/2505.06096v1), 2025-05-09

(11) [Assessing Tenstorrent's RISC-V MatMul Acceleration Capabilities](http://arxiv.org/abs/2505.06085v1), 2025-05-09

(12) [Healthy LLMs? Benchmarking LLM Knowledge of UK Government Public Health Information](http://arxiv.org/abs/2505.06046v1), 2025-05-09

(13) [Short-circuiting Shortcuts: Mechanistic Investigation of Shortcuts in Text Classification](http://arxiv.org/abs/2505.06032v1), 2025-05-09

(14) [Unilogit: Robust Machine Unlearning for LLMs Using Uniform-Target Self-Distillation](http://arxiv.org/abs/2505.06027v1), 2025-05-09

(15) [ArtRAG: Retrieval-Augmented Generation with Structured Context for Visual Art Understanding](http://arxiv.org/abs/2505.06020v1), 2025-05-09

(16) [Exploring the Feasibility of Multilingual Grammatical Error Correction with a Single LLM up to 9B parameters: A Comparative Study of 17 Models](http://arxiv.org/abs/2505.06004v1), 2025-05-09

(17) [Task-Adapter++: Task-specific Adaptation with Order-aware Alignment for Few-shot Action Recognition](http://arxiv.org/abs/2505.06002v1), 2025-05-09

(18) [NeoQA: Evidence-based Question Answering with Generated News Events](http://arxiv.org/abs/2505.05949v1), 2025-05-09

(19) [Summarisation of German Judgments in conjunction with a Class-based Evaluation](http://arxiv.org/abs/2505.05947v1), 2025-05-09

(20) [Elastic Weight Consolidation for Full-Parameter Continual Pre-Training of Gemma2](http://arxiv.org/abs/2505.05946v1), 2025-05-09

(21) [CAPE: Context-Aware Prompt Perturbation Mechanism with Differential Privacy](http://arxiv.org/abs/2505.05922v1), 2025-05-09

(22) [Evolutionary ecology of words](http://dx.doi.org/10.1109/ALIFE-CIS64968.2025.10979831), 2025-05-09

(23) [AgentXploit: End-to-End Redteaming of Black-Box AI Agents](http://arxiv.org/abs/2505.05849v1), 2025-05-09

(24) [Augmented Body Communicator: Enhancing daily body expression for people with upper limb limitations through LLM and a robotic arm](http://arxiv.org/abs/2505.05832v1), 2025-05-09

(25) [Tell Me Who Your Students Are: GPT Can Generate Valid Multiple-Choice Questions When Students' (Mis)Understanding Is Hinted](http://arxiv.org/abs/2505.05815v1), 2025-05-09

(26) [3D CAVLA: Leveraging Depth and 3D Context to Generalize Vision Language Action Models for Unseen Tasks](http://arxiv.org/abs/2505.05800v1), 2025-05-09

(27) [What Is Next for LLMs? Next-Generation AI Computing Hardware Using Photonic Chips](http://arxiv.org/abs/2505.05794v1), 2025-05-09

(28) [A Day in Their Shoes: Using LLM-Based Perspective-Taking Interactive Fiction to Reduce Stigma Toward Dirty Work](http://dx.doi.org/10.1145/3715275.3732090), 2025-05-09

(29) [Sparse Attention Remapping with Clustering for Efficient LLM Decoding on PIM](http://arxiv.org/abs/2505.05772v1), 2025-05-09

(30) [Multi-Agent Systems for Robotic Autonomy with LLMs](http://arxiv.org/abs/2505.05762v1), 2025-05-09

(31) [APOLLO: Automated LLM and Lean Collaboration for Advanced Formal Reasoning](http://arxiv.org/abs/2505.05758v1), 2025-05-09

(32) [Evolutionary thoughts: integration of large language models and evolutionary algorithms](http://arxiv.org/abs/2505.05756v1), 2025-05-09

(33) [Harnessing LLMs Explanations to Boost Surrogate Models in Tabular Data Classification](http://arxiv.org/abs/2505.05744v1), 2025-05-09

(34) [Multimodal Integrated Knowledge Transfer to Large Language Models through Preference Optimization with Biomedical Applications](http://arxiv.org/abs/2505.05736v1), 2025-05-09

(35) [Understanding Stragglers in Large Model Training Using What-if Analysis](http://arxiv.org/abs/2505.05713v1), 2025-05-09

(36) [LLM-Text Watermarking based on Lagrange Interpolation](http://arxiv.org/abs/2505.05712v1), 2025-05-09

(37) [Assessing Robustness to Spurious Correlations in Post-Training Language Models](http://arxiv.org/abs/2505.05704v1), 2025-05-09

(38) [Fine-Tuning Video-Text Contrastive Model for Primate Behavior Retrieval from Unlabeled Raw Videos](http://arxiv.org/abs/2505.05681v1), 2025-05-08

(39) [From Bias To Improved Prompts: A Case Study of Bias Mitigation of Clone Detection Models](http://arxiv.org/abs/2505.05679v1), 2025-05-08

(40) [InstanceGen: Image Generation with Instance-level Instructions](http://arxiv.org/abs/2505.05678v1), 2025-05-08

(41) [Lost in OCR Translation? Vision-Based Approaches to Robust Document Retrieval](http://arxiv.org/abs/2505.05666v1), 2025-05-08

(42) [Adaptive Stress Testing Black-Box LLM Planners](http://arxiv.org/abs/2505.05665v1), 2025-05-08

(43) [Not Like Us, Hunty: Measuring Perceptions and Behavioral Effects of Minoritized Anthropomorphic Cues in LLMs](http://dx.doi.org/10.1145/3715275.3732045), 2025-05-08

(44) [Looking Beyond Language Priors: Enhancing Visual Comprehension and Attention in Multimodal Models](http://arxiv.org/abs/2505.05626v1), 2025-05-08

(45) [CityNavAgent: Aerial Vision-and-Language Navigation with Hierarchical Semantic Planning and Global Memory](http://arxiv.org/abs/2505.05622v1), 2025-05-08

(46) [LiteLMGuard: Seamless and Lightweight On-Device Prompt Filtering for Safeguarding Small Language Models against Quantization-induced Risks and Vulnerabilities](http://arxiv.org/abs/2505.05619v1), 2025-05-08

(47) [Leveraging Large Language Models for enzymatic reaction prediction and characterization](http://arxiv.org/abs/2505.05616v1), 2025-05-08

(48) [scDrugMap: Benchmarking Large Foundation Models for Drug Response Prediction](http://arxiv.org/abs/2505.05612v1), 2025-05-08

(49) [HiBayES: A Hierarchical Bayesian Modeling Framework for AI Evaluation Statistics](http://arxiv.org/abs/2505.05602v1), 2025-05-08

(50) [Enhancing Large Language Models with Faster Code Preprocessing for Vulnerability Detection](http://arxiv.org/abs/2505.05600v1), 2025-05-08

(51) [PRIMG : Efficient LLM-driven Test Generation Using Mutant Prioritization](http://arxiv.org/abs/2505.05584v1), 2025-05-08

(52) [KG-HTC: Integrating Knowledge Graphs into LLMs for Effective Zero-shot Hierarchical Text Classification](http://arxiv.org/abs/2505.05583v1), 2025-05-08

(53) [Generating Physically Stable and Buildable LEGO Designs from Text](http://arxiv.org/abs/2505.05469v1), 2025-05-08

(54) [StreamBridge: Turning Your Offline Video Large Language Model into a Proactive Streaming Assistant](http://arxiv.org/abs/2505.05467v1), 2025-05-08

(55) [ComPO: Preference Alignment via Comparison Oracles](http://arxiv.org/abs/2505.05465v1), 2025-05-08

(56) [Bring Reason to Vision: Understanding Perception and Reasoning through Model Merging](http://arxiv.org/abs/2505.05464v1), 2025-05-08

(57) [UKElectionNarratives: A Dataset of Misleading Narratives Surrounding Recent UK General Elections](http://arxiv.org/abs/2505.05459v1), 2025-05-08

(58) [Conversational Process Model Redesign](http://arxiv.org/abs/2505.05453v1), 2025-05-08

(59) [clem:todd: A Framework for the Systematic Benchmarking of LLM-Based Task-Oriented Dialogue System Realisations](http://arxiv.org/abs/2505.05445v1), 2025-05-08

(60) [GesPrompt: Leveraging Co-Speech Gestures to Augment LLM-Based Interaction in Virtual Reality](http://arxiv.org/abs/2505.05441v1), 2025-05-08

(61) [EcoAgent: An Efficient Edge-Cloud Collaborative Multi-Agent Framework for Mobile Automation](http://arxiv.org/abs/2505.05440v2), 2025-05-08

(62) [Ultra-FineWeb: Efficient Data Filtering and Verification for High-Quality LLM Training Data](http://arxiv.org/abs/2505.05427v1), 2025-05-08

(63) [LiTransProQA: an LLM-based Literary Translation evaluation metric with Professional Question Answering](http://arxiv.org/abs/2505.05423v2), 2025-05-08

(64) [Crosslingual Reasoning through Test-Time Scaling](http://arxiv.org/abs/2505.05408v1), 2025-05-08

(65) [Frame In, Frame Out: Do LLMs Generate More Biased News Headlines than Humans?](http://arxiv.org/abs/2505.05406v1), 2025-05-08

(66) [DSDrive: Distilling Large Language Model for Lightweight End-to-End Autonomous Driving with Unified Reasoning and Planning](http://arxiv.org/abs/2505.05360v1), 2025-05-08

(67) [Hearing and Seeing Through CLIP: A Framework for Self-Supervised Sound Source Localization](http://arxiv.org/abs/2505.05343v1), 2025-05-08

(68) [FLAM: Frame-Wise Language-Audio Modeling](http://arxiv.org/abs/2505.05335v1), 2025-05-08

(69) [ICon: In-Context Contribution for Automatic Data Selection](http://arxiv.org/abs/2505.05327v1), 2025-05-08

(70) [Toward Reasonable Parrots: Why Large Language Models Should Argue with Us by Design](http://arxiv.org/abs/2505.05298v1), 2025-05-08

(71) [PlaceIt3D: Language-Guided Object Placement in Real 3D Scenes](http://arxiv.org/abs/2505.05288v1), 2025-05-08

(72) [HEXGEN-TEXT2SQL: Optimizing LLM Inference Request Scheduling for Agentic Text-to-SQL Workflow](http://arxiv.org/abs/2505.05286v1), 2025-05-08

(73) [Software Development Life Cycle Perspective: A Survey of Benchmarks for Code Large Language Models and Agents](http://arxiv.org/abs/2505.05283v2), 2025-05-08

(74) [PADriver: Towards Personalized Autonomous Driving](http://arxiv.org/abs/2505.05240v1), 2025-05-08

(75) [Latte: Transfering LLMs` Latent-level Knowledge for Few-shot Tabular Learning](http://arxiv.org/abs/2505.05237v1), 2025-05-08

(76) [ChemRxivQuest: A Curated Chemistry Question-Answer Database Extracted from ChemRxiv Preprints](http://arxiv.org/abs/2505.05232v1), 2025-05-08

(77) [QualBench: Benchmarking Chinese LLMs with Localized Professional Qualifications for Vertical Domain Evaluation](http://arxiv.org/abs/2505.05225v1), 2025-05-08

(78) [Stealthy LLM-Driven Data Poisoning Attacks Against Embedding-Based Retrieval-Augmented Recommender Systems](http://arxiv.org/abs/2505.05196v1), 2025-05-08

(79) [Revealing Weaknesses in Text Watermarking Through Self-Information Rewrite Attacks](http://arxiv.org/abs/2505.05190v1), 2025-05-08

(80) [Biomed-DPT: Dual Modality Prompt Tuning for Biomedical Vision-Language Models](http://arxiv.org/abs/2505.05189v1), 2025-05-08

(81) [MARK: Memory Augmented Refinement of Knowledge](http://arxiv.org/abs/2505.05177v1), 2025-05-08

(82) [FedTDP: A Privacy-Preserving and Unified Framework for Trajectory Data Preparation via Federated Learning](http://arxiv.org/abs/2505.05155v1), 2025-05-08

(83) [Text2Cypher: Data Pruning using Hard Example Selection](http://arxiv.org/abs/2505.05122v1), 2025-05-08

(84) [Enhancing Text2Cypher with Schema Filtering](http://arxiv.org/abs/2505.05118v1), 2025-05-08

(85) [Unveiling Language-Specific Features in Large Language Models via Sparse Autoencoders](http://arxiv.org/abs/2505.05111v1), 2025-05-08

(86) [Multi-agent Embodied AI: Advances and Future Directions](http://arxiv.org/abs/2505.05108v1), 2025-05-08

(87) [A Weighted Byzantine Fault Tolerance Consensus Driven Trusted Multiple Large Language Models Network](http://arxiv.org/abs/2505.05103v1), 2025-05-08

(88) [X-Driver: Explainable Autonomous Driving with Vision-Language Models](http://arxiv.org/abs/2505.05098v1), 2025-05-08

(89) [Reliably Bounding False Positives: A Zero-Shot Machine-Generated Text Detection Framework via Multiscaled Conformal Prediction](http://arxiv.org/abs/2505.05084v1), 2025-05-08

(90) [Performance Evaluation of Large Language Models in Bangla Consumer Health Query Summarization](http://arxiv.org/abs/2505.05070v1), 2025-05-08

(91) [WaterDrum: Watermarking for Data-centric Unlearning Metric](http://arxiv.org/abs/2505.05064v1), 2025-05-08

(92) [CodeMixBench: Evaluating Large Language Models on Code Generation with Code-Mixed Prompts](http://arxiv.org/abs/2505.05063v1), 2025-05-08

(93) [Towards Mitigating API Hallucination in Code Generated by LLMs with Hierarchical Dependency Aware](http://arxiv.org/abs/2505.05057v1), 2025-05-08

(94) [LSRP: A Leader-Subordinate Retrieval Framework for Privacy-Preserving Cloud-Device Collaboration](http://arxiv.org/abs/2505.05031v1), 2025-05-08

(95) [A Reputation System for Large Language Model-based Multi-agent Systems to Avoid the Tragedy of the Commons](http://arxiv.org/abs/2505.05029v1), 2025-05-08

(96) [GenAI in Entrepreneurship: a systematic review of generative artificial intelligence in entrepreneurship research: current issues and future directions](http://arxiv.org/abs/2505.05523v1), 2025-05-08

(97) [Scalable Multi-Stage Influence Function for Large Language Models via Eigenvalue-Corrected Kronecker-Factored Parameterization](http://arxiv.org/abs/2505.05017v1), 2025-05-08

(98) [The Pitfalls of Growing Group Complexity: LLMs and Social Choice-Based Aggregation for Group Recommendations](http://dx.doi.org/10.1145/3708319.3733659), 2025-05-08

(99) [Rethinking Invariance in In-context Learning](http://arxiv.org/abs/2505.04994v1), 2025-05-08

(100) [Latent Preference Coding: Aligning Large Language Models via Discrete Latent Codes](http://arxiv.org/abs/2505.04993v1), 2025-05-08
